scripts/hostcfgd

```diff
@@ -781,11 +781,29 @@ class AaaCfg(object):
                 interface_ip = ipv4_addr
         return interface_ip
 
+
+    def check_file_not_empty(self, filename):
+        exists = os.path.exists(filename)
+        if not exists:
+            syslog.syslog(syslog.LOG_ERR, "file size check failed: {} is missing".format(filename))
+            return
+
+        # du command result is: 'size\tpath'
```

Comment:
># du command result is: 'size\tpath' [](http://example.com/codeflow?start=8&length=36)

The comment is not relevant any more. Please remove it.

Reply:
Fixed.